docs/products/clickhouse/howto/create-dictionary.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Create dictionaries in Aiven for ClickHouse®
+sidebar_label: Create dictionary
+---
+
+Create dictionaries in Aiven for ClickHouse® to accelerate queries for better efficiency and performance.
+
+## Dictionaries in Aiven for ClickHouse
+
+A dictionary is a key -> attribute mapping useful for low latent lookup queries, when you
+often access reference lists to retrieve attributes from a key. Dictionary data resides
+fully in memory and is accessed using functions, which are faster than regular SQL queries.
```

Comment:
Since a recent version of clickhouse, you don't need to use the functions `dictGet` or similar, you can just use the dictionary as a table.

Could rewrite as:

> Dictionary data resides fully in memory and for JOINS are often much faster than using a MergeTree table.